common-moonmage.lic

```diff
@@ -45,7 +45,18 @@ module DRCMM
   end
 
   def peer_telescope
-    DRC.bput('peer telescope', 'The pain is too much', 'You learned something useful from your observation', 'Clouds obscure the sky', "While the sighting wasn't quite", 'You peer aimlessly through your telescope', 'You have not pondered your last observation sufficiently', 'You see nothing regarding the future', "You believe you've learned all that you can about", 'Too many futures cloud your mind - you learn nothing.', 'open it', 'You see nothing regarding the future', 'you still learned more', 'Roundtime')
+    DRC.bput('peer telescope', 'The pain is too much',\
+                               'You learned something useful from your observation',\
```

Comment:
💡 You could pull many of these messages from base-constellations instead of duplicating them

Reply:
👍